benchmarks/image_segmentation/tensorflow/3d_unet/README.md

```diff
@@ -50,7 +50,7 @@ modes/precisions:
         --in-graph /home/myuser/pretrained_models/tumor_segmentation_model.h5 \
         --data-location /home/myuser/3dunet_data/brats \
         --batch-size 1 \
-        --socket-id 0 \
+        --socket-id 0
```

Comment:
@ashahba except this one change. can you please undo all other changes. 
anything comes after `--` with `space` are extra args required for the model.

Reply:
Thanks @karthikvadla what about this one:
https://github.com/IntelAI/models/pull/6/files#diff-2a1d8f448567933e0ddf102b2c09903fR221

Reply:
@ashahba Those spaces are intentional. 
Those are all model-specific args (one offs) that are not part of the regular arg parser, so they are defined after the ` -- ` (with a space after the `--`).